### ISSUE TITLE
revoir aussi les règles de contribution

### ISSUE THREAD

**Issue by zeroheure** [mjourdan/linuxfr-design#1]
(je suis modérateur)
Le problème identifié des baisses de contributions ne se réduit pas au design. Il y aurait même contradiction entre le design, la volonté d'augmenter les contributions, et les règles actuelles. Par exemple, le design avec un petit chapeau d'article ne colle pas à la réalité des premières parties de dépêches qui veulent souvent trop en dire.
En outre, et c'est complémentaire, l'exigence actuelle de qualité et de longueur des articles sur linuxfr est à la fois décourageante pour les nouveaux auteurs et peu compatible avec l'envie de suivre l'actualité. Sauf à avoir une solide équipe de rédacteurs bénévole (pourquoi pas ?), ces deux exigences apauvrissent le site.
Enfin, linuxfr s'adresse à différents publics dans divers domaines : certains articles sont pointus, d'autre visent les débutants. Ainsi la qualité de rédaction doit s'adapter au public attendu sur l'article (il faudrait d'ailleurs indiquer le niveau de lecture, ça éviterait probablement une partie des commentaires désobligeant).

---

La qualité et la longueur sont perçues comme allant de pair, alors que ce n'est pas vrai. Il y a parfois peu à dire, d'autrefois l'auteur s'enlise dans le verbiage et les phrases lourdes. Or, la modération rejette ou renvoie en rédaction des articles jugés « trop court », locution qui revient souvent dans la tribune entre modérateurs. De même, ce message automatique de rejet qui implique que longueur = qualité :

> La dépêche que vous avez proposée a été considérée trop courte pour être validée sur LinuxFr.org. En effet, nous n'acceptons que des dépêches de plusieurs lignes, comme celles qui passent sur le site habituellement.

Pourtant les [règles de modération](http://linuxfr.org/regles_de_moderation), accessibles à tous, ne rendent pas visible cette exigence de longueur :

> Rejeter les dépêches non rédigées
Une dépêche doit être rédigée, un minimum étoffée, fournir un ou plusieurs liens, introduire le contexte, résumer la situation, etc. Celles trop courtes ou trop mal rédigées seront rejetées.

Les règles ne parlent pas non plus de qualité rédactionnelle. Si on n'attend pas de dépêche rédigée comme un pro on peut tout de même demander une phase de relectures sur les lourdeurs et la clarté de l'explication, elle même souvent liée à l'articulation des phrases et des idées. On passera sur la clarté de l'énoncé (précision des mots), qui est plus difficile à atteindre. De toute façon, cette clarté est souvent obtenue dans les commentaires.

Dans ma barque et mon moulin j'amène quatre interventions récentes :

1. Une petite dépêche annonçant la mise à disposition d'un aide-mémoire d'administration système. L'auteur est un professionnel expérimenté. La dépêche a été rejetée parce qu'elle était trop brève (et aussi par peur de commentaires virulents contre l'auteur).
2. Une longue dépêche qui voulait trop bien faire, sur les logiciels artistiques sous Linux. Abandonnée pendant un an par ses auteurs, on vient d'en publier la partie sur le graphisme. J'ai du soigneusement reprendre l'exposé sur la colorimétrie et les sondes de calibrage qui était confus et très peu compréhensible pour les débutants (public visé par la dépêche).
3. Deux sondages : « Comment vous inciter à contribuer plus souvent à LinuxFr.org ? » paru le 12/10, 838 votes, 105 commentaires ; et « La modération a posteriori des contenus et commentaires problématiques sur LinuxFr.org » paru le 2/10, 784 votes, 185 commentaires. 
Ces sondages publiés pendant 15 jours ont réuni peu de votes, à l'image au fond de la faible participation sur le site (tandis que le sondage suivant : « Joueur ou non joueur » paru le 29/11 en a réuni le double en 5 jours de publication).
Les commentaires ont été peu suivis et discutés au sein de l'équipe IMHO. Or beaucoup demandaient le retour des brêves et discutaient du caractère intimidant de linuxfr.

C'est déjà trop long, pourtant il y a encore à dire :

- sans exigences rédactionnelle des modérateurs on ne peut éviter le problème récurrent des engueulades en commentaire (qui ergottent souvent sur des détails) ; c'est comme montrer l'exemple. Un tag débutant, avancé, etc. peut aider.
- une dépêche courte peut susciter de longs commentaires de qualité. On a cru pouvoir retrouver cette qualité dans les dépêches grâce à l'espace de rédaction. En fait les commentateurs ne sont pas les rédacteurs : par exemple PBPG ou Zenitram commentent mais ne rédigent pas ; Ariasuni s'est vue accuser de ne proposer que des contributions sans intéret, en référence à ses commentaires, alors qu'elle a été une grosse contributrice en rédaction, personne n'a relevé parce que beaucoup de commentateurs ne fréquentent pas l'espace de rédaction.
- des fidèles visiteurs disparaissent : en déplaçant la référence des commentaires vers la dépêche, on n'incite pas les mordus de faut niveau à commenter sur une dépêche.

**Comment by zeroheure** [comment]
Ouh là la, j'ai l'impression d'avoir abandonné des idées en train... c que j'ai écris est un peu décousu.

**Comment by nono** [comment]
@zeroheure je ne suis pas sûr de voir où tu veux en venir. Tu proposes de reviser les textes (mails, règles de modérations, etc.) pour ne pas décourager les contenus courts ?

**Comment by zeroheure** [comment]
@nono Oui, mais pas seulement. Sans aller trop loin dans les débats (ni faire de révolution!), on doit se mettre d'accord sur quelques changements demandés dans les sondages et ça et là dans les commentaires : 

On devra discuter des commentaires postés sur les sondages, et être plus à l'écoute des désirs des lecteurs.

On doit aussi se mettre d'accord sur le public des différentes dépêches. Il y en a pour tous les goûts mais aussi pour tous les niveaux. La qualité attendue pour les dépêches varie selon le type de lecteurs : 

- qualité rédactionnelle ; 
- clarté — une dépêche sur Emacs un peu confuse gêne peu ses lecteurs qui sont avancés et comprennent, tandis qu'une dépêche très grand public comme "L’expression graphique sous GNU/Linux" ne peut pas sortir avec un paragraphe très confus sur la colorimétrie ;
- détails techniques — par exemple les dépêches noyau ou Gnome se transforment peu à peu en une liste détaillée de tous les changements, sans explications, ce qui est assez indigeste. Or ceux qui ont besoin de ces détails les trouveront ailleurs : doit-on couper, garder, réclamer plus de parties explicatives, ... ?

Les dépêches de nouveautés sont moins nombreuses, est-ce une conséquence de la longueur demandée ? est-ce bien qu'il y en ait moins?

Avec ces discussions on peut savoir quelle place accorder aux dépêches courtes, aux tutoriels, aux articles fouillés (dont certains sont une belle base technique), etc.

**Comment by mjourdan** [comment]
Le 07/03/2017 à 12:15, zeroheure a écrit :
>
> (je suis modérateur)
>

Ah ! J'imagine que vous avez pas mal de choses à dire :-)

> Le problème identifié des baisses de contributions ne se réduit pas au
> design. Il y aurait même contradiction entre le design, la volonté
> d'augmenter les contributions, et les règles actuelles.
>
Je vois ça comme un tout, les règles de modération font nécessairement
partie du design : au même titre que les (im)possibilités techniques,
elles déterminent ce qu'il est possible voire souhaitable de faire, et
ce qu'on veut dissuader ou empêcher. Certaines règles pourraient être
inscrites directement dans le code du site plutôt que répertoriées sur
le wiki.

Par exemple, si on prend la première règle de modération (Éviter les
doublons), on peut concevoir l'interface de modération de telle sorte
qu'elle permette de visualiser en un clin d'oeil les derniers articles
dont les titres se ressemblent ou qui portent les même tags. Ça rend la
tâche moins fastidieuse, et c'est nettement plus sympa que d'écrire « il
est demandé aux modérateurs de visiter le site _régulièrement_ ».

> Par exemple, le design avec un petit chapeau d'article ne colle pas à
> la réalité des premières parties de dépêches qui veulent souvent trop
> en dire.
>
Justement, c'est l'objectif du gabarit de rédaction sur lequel je
propose que l'on réfléchisse : inciter à accrocher le lecteur, dissuader
de le noyer !

> En outre, et c'est complémentaire, l'exigence actuelle de qualité et
> de longueur des articles sur linuxfr est à la fois décourageante pour
> les nouveaux auteurs et peu compatible avec l'envie de suivre
> l'actualité. Sauf à avoir une solide équipe de rédacteurs bénévole
> (pourquoi pas ?), ces deux exigences apauvrissent le site.
>

Je n'avais pas conscience de cette exigence et il me semble nécessaire
de s'interroger sur les différents formats de contenus que le site
propose. Tu évoquais les brêves par exemple, hé bien pour moi, le seul
mot « dépêche » renvoi plutôt à des articles courts ou de moyenne
longueur, mais certainement pas à un article à rallonge tel qu'on en
voit sur Linux ou Gnome.

Pour le reste, ton commentaire et les références que tu donnes soulèvent
des problématiques que je vais tâcher d'intégrer à ma réflexion.

**Comment by zeroheure** [comment]
> Je vois ça comme un tout,

Je suis entièrement d'accord. Il s'agissait de souligner la nécessaire mise à jour du côté de la modération.
Bravo pour le travail déjà effectué en tout cas, qui est tout-à-fait à propos!
